polkadot/runtime/common/src/auctions/tests.rs

```diff
@@ -0,0 +1,1034 @@
+use super::*;
+use crate::{auctions, mock::TestRegistrar};
+use frame_support::{
+	assert_noop, assert_ok, assert_storage_noop, derive_impl, ord_parameter_types, parameter_types,
+	traits::{EitherOfDiverse, OnFinalize, OnInitialize},
+};
+use frame_system::{EnsureRoot, EnsureSignedBy};
+use pallet_balances;
+use polkadot_primitives::{BlockNumber, Id as ParaId};
+use polkadot_primitives_test_helpers::{dummy_hash, dummy_head_data, dummy_validation_code};
+use sp_core::H256;
+use sp_runtime::{
+	traits::{BlakeTwo256, IdentityLookup},
+	BuildStorage,
+	DispatchError::BadOrigin,
+};
+use std::{cell::RefCell, collections::BTreeMap};
+
+type Block = frame_system::mocking::MockBlockU32<Test>;
+
+frame_support::construct_runtime!(
+	pub enum Test
+	{
+		System: frame_system,
+		Balances: pallet_balances,
+		Auctions: auctions,
+	}
```

Comment:
best practice is probably to split this file one more time:

- mock.rs
- tests.rs

mock.rs will contain:

- construct_runtime
- pallet config
- helper functions

then tests.rs only have `#[test]`

I am not sure if this is really strictly better, but this is the normal approach for other pallets if you look in substrate/frame

Reply:
thx a lot for these comments they are super helpfull !